app_dart/lib/src/request_handlers/refresh_github_commits.dart

```diff
@@ -62,47 +63,49 @@ class RefreshGithubCommits extends ApiRequestHandler<Body> {
 
   @override
   Future<Body> get() async {
-    const String projectId = 'flutter-dashboard';
-    const String dataset = 'cocoon';
-    const String table = 'Checklist';
-
+    final GithubService githubService = await config.createGithubService();
     final GitHub github = await config.createGitHubClient();
     const RepositorySlug slug = RepositorySlug('flutter', 'flutter');
-    final Stream<RepositoryCommit> commits =
-        github.repositories.listCommits(slug);
+    final Stream<Branch> branches = github.repositories.listBranches(slug);
     final DatastoreService datastore = datastoreProvider();
-    final TabledataResourceApi tabledataResourceApi =
-        await config.createTabledataResourceApi();
-    final List<Commit> newCommits = <Commit>[];
-    final List<Map<String, Object>> tableDataInsertAllRequestRows =
-        <Map<String, Object>>[];
+    final File file = File('dev/branch_regexps.txt');
```

Comment:
This will require a new Cocoon deployment to pick up the new file.  I think we'll have to request it from GitHub over http.

Reply:
> nit: against.

fixed. Thanks.

Reply:
Considering moving the file to `flutter` or `engine` repo, will change to http request.

Reply:
Done.